frontend/components/Contribute/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Heading } from '../shared/Heading';
+
+const WantToContribute = ({ data }) => {
+  const { description, img } = data;
+  return (
+    <div className="flex items-start justify-around">
+      <div>
+        <span className="block mt-16 py-5 uppercase">
```

Comment:
this span is not required anymore you can add custom classed to heading component directly ... also take a look about the heading component after changes because some new options are there like uppercase